README.md

```diff
@@ -7,15 +9,41 @@ yarn install
 
 ### Compiles and hot-reloads for development
 ```
-yarn run serve
+yarn serve
 ```
 
 ### Compiles and minifies for production
 ```
-yarn run build
+yarn build
+```
+
+## Start local server
+Starts a local server with the built files. Requires `yarn build` or `yarn heroku-postbuild` to be run for it to work.
+```
+yarn start
 ```
 
 ### Lints and fixes files
 ```
-yarn run lint
+yarn lint
 ```
+## What's built in:
+- Today's weather forecast in Tokyo, Japan
+- Current top 10 HackerNews articles based on votes
+- A famous quote to inspire you
+- The latest TED Talk for some quick learning
+- A love compatibility tester for a bit of fun
```

Comment:
How about changing this to:

A love compatibility tester, because why not?

Reply:
Done!